src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -124,7 +139,7 @@ def __init__(self, transfer_service: Optional[Any] = None):
         self._transfer_service = transfer_service
 
     def upload(self, file_name: str, on_uploaded: Optional[Callable] = None):
-        """Upload a file if it's unavailable on the server
+        """Upload a file if it's unavailable to the server
```

Comment:
The original docstring was better. This new one doesn't make sense with the conditional clause spliced into the middle of the sentence.

Reply:
Done. Thank you.